streampark-console/streampark-console-service/src/main/resources/mapper/core/ApplicationMapper.xml

```diff
@@ -141,6 +141,16 @@
             limit 1
     </select>
 
+    <select id="countAffectedJobsByClusterId" resultType="java.lang.Integer" parameterType="java.lang.Long">
+        select
+            count(1)
+        from t_flink_app
+        where flink_cluster_id = #{clusterId}
+            and state in (5, 7, 9)
+            and (end_time IS NULL or TIMESTAMPDIFF(SECOND, end_time, NOW()) &lt;= 1)
```

Comment:
Does TIMESTAMPDIFF support in h2 and pgsql ?

Reply:
Thank you very much for the review. I have made changes to the code. Could you please review it when you have time.

Reply:
> Does TIMESTAMPDIFF support in h2 and pgsql ?

Hi @wolfboys ，Thank you very much for the review. I have made changes to the code. Could you please review it when you have time.